atlas

```diff
@@ -222,7 +213,6 @@ Will result in the following tree:
         ├── es_419.json
         └── fr_CA.json
 
-More examples are available in the repository docs: https://github.com/openedx/openedx-translations .
 
 
 
```

Comment:
I know this isn't a change from this PR, but it'd probably be good to remove a couple of the newlines here

Reply:
The root-cause is `gengetoptions`. I've removed one new line, but the rest are enforced by `gengetoptions`.